usr/share/rear/prep/default/320_include_uefi_env.sh

```diff
@@ -47,13 +47,21 @@ if grep -qw efivars /proc/mounts; then
     SYSFS_DIR_EFI_VARS=/sys/firmware/efi/efivars
 fi
 
-# next step, is checking /boot/efi directory (we need it)
-if [[ ! -d /boot/efi ]]; then
-    return    # must be mounted
+# next step, is case-sensitive checking /boot for case-insensitive /efi directory (we need it)
+if [[ -n `find /boot -maxdepth 1 -iname efi -type d` ]]; then
```

Comment:
From [coding style](https://github.com/rear/rear/wiki/Coding-Style#code-must-be-easy-to-read)
Use $( COMMAND ) instead of backticks \`COMMAND\`